index.html

```diff
@@ -1,299 +1,246 @@
-            remote work.
-            <br />
-            I've also been a good helping hand to a lot of developers in
-            different communities!
-            <br />
-            if you'd like to get in contact about potential job opportunities,
-            please reach out!
-          </p>
-          <a
-            href="https://docs.google.com/document/d/1lYq0Kb50TujzJzDWGobIchRI17JMnAiZMF9OBQgNU68/edit?usp=sharing"
-            class="link"
-            target="_blank"
-            rel="noreferrer noopener"
-          >
-            Get my resume
-          </a>
-          <img class="about-me-icon" src="./media/svg/icon-2.svg" alt="Icon" />
-        </section>
-        <hr />
-        <div class="about-me-section stacks">
-          <ul>
-            <li class="languages list-title">
-              Languages
-              <ul>
-                <li>JavaScript</li>
-                <li>TypeScript</li>
-                <li>Ruby</li>
-                <li>HTML</li>
-                <li>CSS</li>
-                <li>Sass</li>
-              </ul>
-            </li>
-          </ul>
-          <ul>
-            <li class="frameworks list-title">
-              Frameworks
-              <ul>
-                <li>React & Redux</li>
-                <li>Material-UI</li>
-                <li>Jest</li>
-                <li>Bootstrap</li>
-                <li>Ruby on Rails</li>
-                <li>RSPec</li>
-              </ul>
-            </li>
-          </ul>
-          <ul>
-            <li class="skills list-title">
-              Skills
-              <ul>
-                <li>GitHub</li>
-                <li>CodePen</li>
-                <li>GitLab</li>
-                <li>Terminal</li>
-                <li>WSL</li>
-                <li>Remote Pair-Programming</li>
-                <li>Teamwork</li>
-                <li>Mentoring</li>
-                <li>Code Reviews</li>
-                <li>Mobile/Responsive Development</li>
-              </ul>
-            </li>
-          </ul>
-        </div>
+    </div>
+    <section class="contact" id="contact">
+      <div>
+        <h2 class="contact-description">
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h2>
       </div>
-    </main>
 
-    <script
-      src="https://kit.fontawesome.com/e966a3bd22.js"
-      crossorigin="anonymous"
-    ></script>
-  </body>
-</html>
+      <form action="https://formspree.io/f/xknywqjb" method="POST" id="form">
+        <label for="name">
+          <input type="text" id="name" placeholder="Full name" minlength="2" maxlength="30" name="name" required="">
+        </label>
+        <label for="email">
+          <input type="email" id="email" placeholder="Email address" name="email" required="">
+        </label>
+        <label for="msg">
+          <textarea name="msg" id="msg" cols="30" rows="10" placeholder="Enter text here" maxlength="500"></textarea>
```

Comment:
- [ ] Contact form is looking great! The design is just like the template, however, the user shouldn't be able to submit the form if one or more of the fields is empty. Please make the text area a required field.